pkg/api/collectorcontrollerv1alpha1/types.go

```diff
@@ -437,22 +441,32 @@ const (
 	// nr_periods is number of periods that any thread in the cgroup was runnable
 	NRPeriodsSource Source = "nr_periods"
 
+	AvgNRPeriodsSource Source = "avg_nr_periods"
+
 	// NRThrottledSource is the source for nr_throttled per sec
 	// throttled_time is the total time individual threads within the cgroup were throttled
 	NRThrottledSource Source = "nr_throttled"
 
+	AvgNRThrottledSource Source = "avg_nr_throttled"
+
 	// OOMKillCountSource is the source for OOM Kill counter
 	OOMKillCountSource Source = "oom_kill"
+
+	OOMCountSource Source = "oom"
```

Comment:
Add some docs at least to this one, what's the difference between this source and the one above?

Reply:
Done